setup.cfg

```diff
@@ -1,16 +0,0 @@
-[flake8]
-max-line-length = 88
-ignore = E203, E266, E501, W503
-max-complexity = 18
-select = B,C,E,F,W,T4,B9
-exclude = 
-  .git,
-  build,
-  __pycache__,
-  tests/examples_arguments_syntax,
-  tests/examples_methods_syntax,  
-  altair/vegalite/v?/schema
-
-[metadata]
-description-file = README.md
```

Comment:
Can the metadata seciton here already be removed or should that happen as part of #3007?

Reply:
Valid point. Should be part of #3007. Undelete metadata section.